vulcanexus.repos

```diff
@@ -54,15 +54,15 @@ repositories:
   eProsima/Shapes-Demo:
     type: git
     url: https://github.com/eProsima/ShapesDemo.git
-    version: v2.7.0
+    version: v2.7.1
   eProsima/Shapes-Demo-TypeSupport:
     type: git
     url: https://github.com/eProsima/ShapesDemo-TypeSupport.git
     version: v1.0.0
   eProsima/Vulcanexus-Base:
     type: git
     url: https://github.com/eProsima/vulcanexus.git
-    version: 51756543ac2bf2ff01f355528dbbc01af63979cf
+    version: v2.0.3
```

Comment:
Vulcanexus does not have any tag as of now

Reply:
We know, we have decided to tag them. In fact, we'll create the tag right after merging this PR.